app/serializers/sidebar_section_serializer.rb

```diff
@@ -4,7 +4,7 @@ class SidebarSectionSerializer < ApplicationSerializer
   attributes :id, :title, :links, :slug, :public, :section_type
 
   def links
-    object.sidebar_section_links.map { |link| SidebarUrlSerializer.new(link.linkable, root: false) }
```

Comment:
This is a little wasteful because there is no need for us to create the `sidebar_section_links` records.